### DISCUSSION TITLE
[Bug]: Can't import Engine module

### DISCUSSION THREAD

**Discussion by SilverStarCoder** [open-edge-platform/anomalib#1712] [Q&A]
### Describe the bug

I am following the "001_getting_started" tutorial notebook. After installing anomalib via option 1: `%pip install anomalib`, and then running `from anomalib.engine import Engine`, I get the module not found error: "ModuleNotFoundError: No module named 'anomalib.engine'" .

### Dataset

N/A

### Model

N/A

### Steps to reproduce the behavior

1. Install anomalib: `%pip install anomalib`
2. Run `from anomalib.engine import Engine`

### OS information

OS information: Ubuntu 20.04.6 LTS
Python Version: 3.9.5 
Anomalib Version: 0.7.0

### Expected behavior

The notebook to run without any issues. 

### Pip/GitHub

pip

### Configuration YAML

```yaml
N/A
```

### Logs

```shell
N/A
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by samet-akcay** [comment]
@SilverStarCoder, `Engine` is a new concept that comes with v1. If you are using an old version, anomalib cannot find it. Hope this answers your question

**Reply by SilverStarCoder** [reply]
@samet-akcay , Surely `%pip install anomalib` should be pulling the latest version available? If not, then maybe the notebook should be updated.

**Reply by samet-akcay** [reply]
no unfortunately v1 is not on pypi yet :/ This is due to the security checks of the v1. Maybe we add a note or warning to the readme file

**Reply by samet-akcay** [reply]
if you install anomalib from pypi, you will install v0.7.0 unfortunately

**Reply by SilverStarCoder** [reply]
Ok thank you for the update.

**Reply by samet-akcay** [reply]
we are working hard to pass the security checks so we could release it on pypi.
